src/transforms/filter.js

```diff
@@ -188,9 +200,11 @@ function getDataToCoordFunc(gd, trace, target) {
 
         setConvert(ax);
 
-        // build up ax._categories (usually done during ax.makeCalcdata()
-        for(var i = 0; i < target.length; i++) {
-            ax.d2c(target[i]);
+        if(ax.type === 'category') {
```

Comment:
optimization... though it could have led to spurious confusing "invalid date" console messages if your target values are valid in their own calendar but not in the axis calendar.